phaseshifts/backends.py

```
@@ -164,22 +144,15 @@ def autogen_from_input(self, bulk_file, slab_file, tmp_dir=None, **kwargs):
 
         if _load_viperleed_modules() is not None:
-            raise BackendError(
-                "eeasisss backend requires --slab <POSCAR> when using ViPErLEED."
-            )
+            raise BackendError("eeasisss backend requires --slab <POSCAR> when using ViPErLEED.")
 
-        raise BackendError(
-            "eeasisss backend requires a native EEASiSSS library or "
-            "'phaseshifts[viperleed]'."
-        )
+        raise BackendError("eeasisss backend requires a native EEASiSSS library or " "'phaseshifts[viperleed]'.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)